src/main/java/com/lalabrand/ecommerce/item/ItemController.java

```diff
@@ -15,6 +16,11 @@ public ItemController(ItemService itemService) {
         this.itemService = itemService;
     }
 
+    @QueryMapping(name = "itemsByCategoryId")
+    public List<ItemDto> getItemsByCategoryId(@Argument Integer categoryId) {
```

Comment:
Pls rename it to find items by category( Because we are not using get in method naming)

Reply:
ok